src/System.Text.Encoding/tests/Decoder/Decoder.netcoreapp.cs

```diff
@@ -0,0 +1,57 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Text.Encodings.Tests
+{
+    public partial class DecoderTests
+    {
+        [Fact]
+        public static void GetCharCount_Span_MatchesEncodingCharCount()
+        {
+            const string Text = "hello world";
```

Comment:
nit: I would prefer to avoid using "Text" as it is part of the encoding namespace